web/app/widgets/button/button.settings.tpl.html

```diff
@@ -7,157 +7,239 @@ <h3>Button Settings</h3>
+                                    <item-picker ng-model="form.item"></item-picker>
+                                </div>
                             </div>
-                        </div>
-                    </uib-tab>
 
+                            <section ng-if="form.show_item_value">
+                                <div class="form-group" ng-class="{error: _form.unit.$error && _form.submitted}">
+                                    <label class="control-label col-xs-3">Unit (suffix)</label>
+                                    <div class="col-xs-4">
+                                        <input name="col" ng-disabled="form.useserverformat" ng-model="form.unit" class="form-control" />
+                                    </div>
+                                </div>
+                                <div class="form-group" ng-class="{error: _form.format.$error && _form.submitted}">
+                                    <label class="control-label col-xs-3">Format</label>
+                                    <div class="col-xs-7">
+                                        <input name="col" ng-disabled="form.useserverformat" ng-model="form.format" class="form-control" />
+                                    </div>
+                                    <label class="control-label col-lg-3 col-md-3"></label>
+                                    <div class="col-lg-9 col-md-9">
+                                        <div class="checkbox">
+                                            <label>
+                                                <input type="checkbox" name="vertical" ng-model="form.useserverformat" /> Use server-provided format if available (openHAB 2)
+                                            </label>
+                                        </div>
+                                    </div>
+                                </div>
 
-                </uib-tabset>
-                
+                                <div class="form-group" ng-class="{error: _form.nolinebreak.$error && _form.submitted}">
+                                    <label class="control-label col-lg-3 col-md-3">Layout</label>
+                                    <div class="col-lg-9 col-md-9">
+                                        <div class="checkbox">
+                                            <label>
+                                                <input type="checkbox" name="vertical" ng-model="form.nolinebreak" /> Value right of label instead of below
+                                            </label>
+                                        </div>
+                                    </div>
+                                </div>
+                            </section>
+                        </uib-tab>
+                    </uib-tabset>
                 </div>
-                <div class="col-xs-3" style="height: 80px">
+                <div class="col-xs-3" style="height: 80px" ng-if="form.action_type !== 'navigate'">
```

Comment:
An idea: maybe only hide the second box (active) and change the "Inactive" label to simply "Label"

Reply:
Actually the active concept is still in effect because an item can still be defined when for "navigate" action types (if "show item value" is checked)...

Reply:
Implemented background color change on navigate since it still has to change color when item state changes.
Added active/inactive states to settings page.